admin/acceptance.py

```diff
@@ -984,6 +1017,16 @@ def capture_journal(reactor, host, output_file):
     ran.addCallback(lambda ignored: formatter(b""))
 
 
+def dummy_formatter(output_file):
+    """
+    Create an output handler which turns journald's export format back into
```

Comment:
There's no journald involved? This would also also need to take host and process name as arguments, since unlike journald it's not available in the message itself.